file_paths.md

```diff
@@ -204,6 +217,20 @@ user or group.
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.cfg                     # pxp-agent configuration file
+            modules.d *                       # directory where pxp-agent modules are stored
+                puppet.cfg *                  # ../../var/modules/puppet configuration
+        var *
+            log
+                pxp-agent.log                 # enabled by default
+            modules *
+                puppet.bat *                  # executes Puppet and provides results of Puppet runs
```

Comment:
I changed that after looking at the fixes done by @MikaelSmith to get pxp-agent unit tests working on Windows: https://github.com/puppetlabs/pxp-agent/tree/master/lib/tests/resources/modules.

I'm not sure if this is necessary. @ferventcoder @Iristyle, can you help me here?